code/modules/mob/living/holders.dm

```diff
@@ -140,3 +140,25 @@
 	item_state = "cat1"
 
 	update_itemstate_on_twohand = 1
+
+//SLIMES
+/obj/item/weapon/holder/animal/slime
+	name = "slime holder"
+	desc = "It seeps through your fingers"
+
+/obj/item/weapon/holder/animal/slime/New(loc, mob/M)
+	..()
+	var/mob/living/simple_animal/slime/S = stored_mob
+	icon_state = S.icon_state
+
+/obj/item/weapon/holder/animal/slime/proc/unfreeze()
+	stored_mob.canmove = 1
+	Destroy()
+
+/obj/item/weapon/holder/animal/slime/throw_impact(atom/hit_atom)
+	..()
+	unfreeze()
+
+/obj/item/weapon/holder/animal/slime/attack_hand(mob/user)
+	if(user.get_active_hand() == src) unfreeze()
```

Comment:
This is impossible: attack_hand requires the user's active hand to be empty when clicking. Are you meaning to use attack_self?